### DISCUSSION TITLE
Module Coupons error

### DISCUSSION THREAD

**Discussion by nandogva63** [microweber/microweber#1087] [General]
**Coupon tables are not created during installation**
HTTP 500 Internal Server Error
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'cart_coupons' doesn't exist (Connection: mysql, SQL: select count(*) as aggregate from `_cart_coupons`) (View: /var/www/localhost/htdocs/src/MicroweberPackages/Module/resources/views/admin/view.blade.php)PDOException  QueryException  ViewException_

### Move files from:
userfiles/modules/shop/coupons/src/database/migrations/2023_00_00_000000_create_cart_coupons_table.php
userfiles/modules/shop/coupons/src/database/migrations/2023_00_00_000001_create_cart_coupons_log_table.php
to
src/MicroweberPackages/Shop/migrations

**Comment by nandogva63** [comment]
Move these files before start web install 
After Web install, if you have shell access, run the command:php artisan migrate --`path=userfiles/modules/shop/coupons/src/database/migrations`

**Comment by peter-mw** [comment]
hi. it should be fixed now
